index.js

```diff
@@ -2,61 +2,54 @@
 
 import eslint from './src/eslint.js';
 import * as fs from 'fs';
+import {createFromEslintResult, getFilteredMessages} from './src/baseline.js';
 
 const FILE_NAME = '.eslint-baseline.json';
 
-async function exec(){
-    
-    if(!fs.existsSync(FILE_NAME)){
+async function exec() {
+    if (!fs.existsSync(FILE_NAME)) {
         console.log('baseline not found attempting to create...');
-        let result = await eslint.execute();
-        fs.appendFileSync(FILE_NAME, JSON.stringify(result, null, 4) );
+
+        const result = await eslint.execute();
+        const baseline = createFromEslintResult(result);
```

Comment:
@ChobotX If the `.eslint-baseline.json` file doesn't exist it will get to `src/baseline.js:14:28` and fail with
```
TypeError: file.split is not a function
```

Reply:
Oh, `createFromEslintResult` should never touch the `.eslint-baseline.json`.
The error you see seems like that eslint has returned a file path that you cannot reach with `fs.readFileSync`.

Weird, coul'd you replicate/debug it please? I am not having this issue 🤔 

Skipping the file in this case is a bad approach, I need to identify the problem first.

Reply:
My bad, had wrong version, found, fixed.. thanks 👍